scripts/react_native_pods.rb

```diff
@@ -646,5 +646,6 @@ def __apply_Xcode_12_5_M1_post_install_workaround(installer)
   #   "Time.h:52:17: error: typedef redefinition with different types"
   # We need to make a patch to RCT-Folly - remove the `__IPHONE_OS_VERSION_MIN_REQUIRED` check.
   # See https://github.com/facebook/flipper/issues/834 for more details.
-  `sed -i -e  $'s/ && (__IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_10_0)//' Pods/RCT-Folly/folly/portability/Time.h`
+  time_header = "#{Pod::Config.instance.installation_root.to_s}/Pods/RCT-Folly/folly/portability/Time.h"
```

Comment:
Ha of course! My hack lives on :laughing: but I didn't think about making it durable.
Is there an RCT-Folly issue pursuing this for a non-hack fix? Just curious

Reply:
Would the non hack fix not be https://github.com/facebook/folly/pull/1688 ?

Reply:
@Saadnajmi that looks like it exactly! Hopefully it lands soon-ish and react-native 0.68 can include it